setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 160
```

Comment:
Isn't it 79 for black?

Reply:
I change line-length for black to 140 in pyproject.toml and for flake8 max-line-length to 140